packages/react-examples/src/react-tabs/__snapshots__/Pivot.AlwaysRender.Example.tsx.shot

```diff
@@ -46,6 +46,7 @@ exports[`Component Examples renders Pivot.AlwaysRender.Example.tsx correctly 1`]
           }
       htmlFor="Toggle0"
       id="Toggle0-label"
+      onClick={[Function]}
```

Comment:
Why is this appearing in so many snapshots?

Reply:
fixed!